### PR TITLE
Add task params to report

### PR DESCRIPTION
**Pull request by mirpedrol**
Add task specific params to the report table.
- Added time, cpus, powerdrawCPU, cpuUsage and memory
- Modify JS unit conversion functions to work as the groovy helper functions

Example (raw values)
image
Example (human readable)
image
- I classified them as metadata, as metrics I left CO2 emissions and energy consumption which are the final metrics

### CODE REVIEW

File: plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2Record.groovy [hunk 1]
```diff
@@ -30,7 +30,6 @@ class CO2Record extends TraceRecord {
         this.energy = energy
         this.co2e = co2e
         this.time = time
-        this.cpus = cpus
         this.powerdrawCPU = powerdrawCPU
         this.cpuUsage = cpuUsage
```
Comment (skrakau) [inline plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2Record.groovy:34]: Why is this removed? 
It seems I forgot to add it to `this.store` below, but wouldn't it be needed also for the report table?

Comment (mirpedrol) [reply plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2Record.groovy:34]: Ups that was unintentional, I only wanted to remove the duplicated co2e line below, thanks for spotting it!

**Review by skrakau — APPROVED** [review]
Great, LGTM :)
